registration/models.py

```diff
@@ -83,14 +83,14 @@ class BetaTestApplication(ValidateModelMixin, TimeStampedModel):
     )
     instance_name = models.CharField(
         max_length=255,
-        help_text=('The name of your institution, company or project.'
-                   '\n\nExample: Hogwarts Online Learning'),
+        help_text=('The name of your institution, company or project.<br/>'
+                   'Example: Hogwarts Online Learning'),
```

Comment:
For line breaks yes, we will just have to add 'white-space: pre-line' to
the css instead. But one of them contains a link, we have to use html for
that.